tests/cli/open_package/test_check.py

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import os
+
+import params
+
+"""
+Test command "open-package --check"
+"""
+
+
+def test_check_specific_package_with_warnings(cli):
+    """Test checking a specific package that has warnings."""
+    library = params.POPULATED_LIBRARY
+    cli.add_library(library.dir)
+
+    # Check the TO220AB package
+    pkg_path = cli.abspath(
+        os.path.join(library.dir, "pkg", "0eaf289c-166d-4bd9-a4ba-dbf6bbc76ef1")
+    )
+    code, stdout, stderr = cli.run("open-package", "--check", pkg_path)
+
+    # Should complete successfully even with warnings
+    assert stdout == f"Open package '{pkg_path}'...\nFinished with errors!\n"
```

Comment:
Here I miss the summary output, like we have it for ERC/DRC:

```
Run ERC...
  Approved messages: 7
  Non-approved messages: 2
```

See my corresponding comment in the cpp code.